### PR TITLE
Travis CI integration

### PR DESCRIPTION
**Pull request by acrellin**

### CODE REVIEW

**Comment by acrellin** [comment]
:heavy_check_mark: :green_heart:

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    - nginx
+    - xvfb
+    - chromium-browser
+    - unzip
+    - libnss3
+    - libgconf-2-4
+  postgresql: "9.4"
+
+notifications:
+  webhooks:
```
Comment (stefanv) [inline .travis.yml]: Want to hook this into Slack instead?

File: .drone/travis_install.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -ex
+
+
+section "install.base.requirements"
+pip install --upgrade pip
+hash -d pip  # find upgraded pip
+pip install --retries 3 -q requests six python-dateutil nose nose-exclude mock
```
Comment (bnaul) [inline .drone/travis_install.sh]: Do we still use `nose`? I think we can remove that and `mock`.

File: .travis.yml (hunk 1 above)
Comment (acrellin) [inline .travis.yml]: Ah, right

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  - source .drone/travis_funcs.sh
+  - export FLASK_APP=cesium_app/flask_server.py
+
+install:
+  - .drone/travis_install.sh
+
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
```
Comment (stefanv) [inline .travis.yml:94]: Is this necessary?

File: .drone/travis_install.sh (hunk 2 above)
Comment (acrellin) [inline .drone/travis_install.sh]: :+1:

File: public/scripts/main.js [hunk 4]
```diff
@@ -8,7 +8,6 @@ var Tabs = ReactTabs.Tabs;
 var TabList = ReactTabs.TabList;
 var TabPanel = ReactTabs.TabPanel;
 
-import _ from 'underscore'
```
Comment (bnaul) [inline public/scripts/main.js:11]: lol what #javascript

File: .travis.yml (hunk 3 above)
Comment (acrellin) [inline .travis.yml:94]: This is straight out of the Travis docs :)

**Comment by bnaul** [comment]
:+1:

File: tools/frontend_tests.py [hunk 5]
```diff
@@ -34,7 +34,8 @@ def clear_db():
 if __name__ == '__main__':
     add_test_yaml()
     clear_db()
-    web_client = subprocess.Popen(['make', 'debug'], cwd=base_dir)
+    env = os.environ.copy()
```
Comment (stefanv) [inline tools/frontend_tests.py]: Not needed--default behavior

File: .drone/travis_install.sh [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+section "init.cesium_web"
+make paths
+sudo make db_init
+make db_test_data
+make bundle
+section_end "init.cesium_web"
+
+
+section "install.chromedriver"
```
Comment (stefanv) [inline .drone/travis_install.sh:35]: Cache this download?

File: .drone/travis_install.sh (hunk 2 above)
Comment (stefanv) [inline .drone/travis_install.sh]: None of these may be necessary any more.  Perhaps requests still used now before tornado refactor.

**Comment by stefanv** [comment]
Looking good!

File: .travis.yml (hunk 3 above)
Comment (stefanv) [inline .travis.yml:94]: Yes, but is it necessary ;)

**Comment by stefanv** [comment]
:fire:

**Pull request merged**